code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -81,6 +81,16 @@
 	. = ..()
 	AddComponent(/datum/component/squeak, list('sound/effects/clownstep1.ogg'=1,'sound/effects/clownstep2.ogg'=1), 50)
 
+/obj/item/clothing/shoes/clown_shoes/equipped(mob/user, slot)
+	GET_COMPONENT_FROM(mood, /datum/component/mood, user)
+	if(mood)
+		mood.add_event("noshoes", /datum/mood_event/noshoes)
```

Comment:
why do you get a negative event for having the clown shoes equipped? isn't this backwards.

Reply:
uhh woops good catch

Reply:
besides that, i should probably make sure the person is actually a clown lol